cookbook/validation/custom_constraint.rst

```diff
@@ -189,21 +189,14 @@ tag and an ``alias`` attribute:
             ->register('validator.unique.your_validator_name', 'Fully\Qualified\Validator\Class\Name')
             ->addTag('validator.constraint_validator', array('alias' => 'alias_name'));
 
-Your constraint class should now use this alias to reference the appropriate
-validator::
+As mentioned above, Symfony will automatically look for a class named after
+the constraint, with ``Validator`` appended. You can override this in your constraint class::
 
     public function validatedBy()
     {
-        return 'alias_name';
+        return 'Fully\Qualified\Class\Named\ConstraintValidator'; // or 'alias_name' if provided
```

Comment:
I wanted to make it more noticeable that the class name may not be the constraint FQCN with `Validator` appended, what do you think about `Fully\Qualified\ConstraintValidator\Class\Name`?